protokube/pkg/protokube/volume_mounter.go

```diff
@@ -188,7 +188,7 @@ func (k *VolumeMountController) safeFormatAndMount(volume *Volume, mountpoint st
 
 		if mountedDevice != "" {
 			if mountedDevice != source {
-				return fmt.Errorf("device already mounted at %s, but is %s and we want %s", target, mountedDevice, source)
+				glog.Warningf("container mounts may not work: device already mounted at %s, but is %s and we want %s", target, mountedDevice, source)
```

Comment:
Was causing issues since mountedDevice was named `/dev/sda` but the source was something like `/rootfs/mnt/master-41d7b6d1-2af5-11e8-ae20-0242ac113a07` which is technically correct. I can remove it in this PR if you think it'll cause issues in the upcoming release.